content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -44,6 +44,12 @@ Hyper-V Requirements:     VM Monitor Mode Extensions: Yes
                           Data Execution Prevention Available: Yes
 ```
 
+If you see the following output, your system already has a Hypervisor installed and you can skip the next step.
```

Comment:
Hi @jarst 
How about wrapping this in `{{< note >}}` / `{{< /note >}}` shortcode block, so it renders as a note?

Reply:
Hi @sftim 
I could do that. The question is does:
![image](https://user-images.githubusercontent.com/846521/59186056-3a2bea80-8b72-11e9-813c-9a426e300c90.png) 
looks better to you, than this: 
https://deploy-preview-14776--kubernetes-io-master-staging.netlify.com/docs/tasks/tools/install-minikube/#install-a-hypervisor 
?

Reply:
I like the look of the tabbed version; in fact, I've got PR #14013 that tries something similar.